packages/shared/src/lib/auxiliary/transak/interfaces/transak-api-fiat-currencies-response.interface.ts

```diff
@@ -0,0 +1,46 @@
+import { FiatCurrency } from '@core/market'
+
+export interface ITransakApiFiatCurrenciesResponse {
+    response: ITransakApiFiatCurrenciesResponseItem[]
+}
+
+export interface ITransakApiFiatCurrenciesResponseItem {
+    symbol: [FiatCurrency]
```

Comment:
is this intended to be of type Array with exactly one element?

Reply:
No, it was intended to be `keyof typeof FiatCurrency`, now it is fixed!